contiamo/resources.py

```diff
@@ -1,204 +1,206 @@
+import logging
+
 from contiamo.http_client import HTTPClient
 from contiamo.utils import raise_response_error, parse_query_result
 
-import logging
+
 logger = logging.getLogger(__name__)
 
 
 def CreateNestedResource(base_class, parent, **kwargs):
-  class_name = type(parent).__name__ + base_class.__name__.rstrip('Resource')
-  properties = {'parent': parent}
-  properties.update(kwargs)
-  return type(class_name, (base_class,), properties)
+    class_name = type(parent).__name__ + base_class.__name__.rstrip('Resource')
+    properties = {'parent': parent}
+    properties.update(kwargs)
+    return type(class_name, (base_class,), properties)
 
 
 ###
 # Main client
 ###
-class Client:
+class Client(object):
```

Comment:
Isn't that Python 2 syntax? Is this part of an explicit effort to make it Python 2.6 compatible?

Reply:
nope, just old habit since I'm used to writing Python 2. I'll take these out.